microservice-app/src/main/java/com/sdevelopment/skt/microservice/dao/impl/ProductRepositoryImpl.java

```diff
@@ -0,0 +1,84 @@
+package com.sdevelopment.skt.microservice.dao.impl;
+
+import com.sdevelopment.skt.common.domain.Product;
+import com.sdevelopment.skt.microservice.dao.ProductRepository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.StoredProcedureQuery;
+import java.util.List;
+
+public class ProductRepositoryImpl implements ProductRepository {
```

Comment:
If you implement the `ProductRepository` interface, you not only have to implement your custom methods, also all the methods corresponding to `CrudRepository` (save, findOne, exists, etc) and that should not be necessary

Reply:
Problem solved after refactoring those classes.